packages/office-ui-fabric-react/src/components/Persona/Persona.base.tsx

```diff
@@ -31,12 +31,18 @@ export class PersonaBase extends BaseComponent<IPersonaProps, {}> {
   }
 
   public render(): JSX.Element {
+    // wrappring default render behavior based of various this.props properties
+    const _onRenderPrimaryText = this._onRenderText(this._getText()),
+      _onRenderSecondaryText = this._onRenderText(this.props.secondaryText),
+      _onRenderTertiaryText = this._onRenderText(this.props.tertiaryText),
+      _onRenderOptionalText = this._onRenderText(this.props.optionalText);
+
     const {
       hidePersonaDetails,
-      onRenderOptionalText,
-      onRenderPrimaryText,
-      onRenderSecondaryText,
-      onRenderTertiaryText
+      onRenderOptionalText = _onRenderOptionalText,
+      onRenderPrimaryText = _onRenderPrimaryText,
+      onRenderSecondaryText = _onRenderSecondaryText,
+      onRenderTertiaryText = _onRenderTertiaryText
     } = this.props;
```

Comment:
This means that we immediately override the passed in onRender*Text props, so they will no longer be called if passed in

Reply:
Not exactly. These are like the default value to take in case nothing is passed in `this.props`. Each of the callbacks will take the value passed to them via `this.props`, if nothing is passed for a particular callback then the default value ( `_onRender*Text` ) will be assigned to them (which is same implementation like earlier).

So if there exists a property as `this.props.onRenderPrimaryText` then this value will be assigned to `onRenderPrimaryText` insted of `_onRenderPrimaryText`.

Reply:
Ah, gotcha, I misread. Thanks for clarifying!